guide/samples/tests/mocking_lros.rs

```diff
@@ -0,0 +1,246 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Examples showing how to mock a client in tests.
+
+// ANCHOR: all
+#[cfg(test)]
+mod test {
+    use google_cloud_gax as gax;
+    use google_cloud_longrunning as longrunning;
+    use google_cloud_speech_v2 as speech;
+    use google_cloud_wkt as wkt;
```

Comment:
Consider more `use` declarations to make this code easier to follow.

```suggestion
    use google_cloud_gax as gax;
    use google_cloud_longrunning as longrunning;
    use longrunning::mode::{GetOperationRequest, Operation};
    use google_cloud_speech_v2 as speech;
    use speech::{BatchRecognizeRequest, BatchRecognizeResponse, OperationMetadata};
    use google_cloud_wkt as wkt;
```

Reply:
Done, and it is nicer.

I was a little concerned about hiding which types are in `longrunning` vs. `speech`. (e.g. `Operation` vs. `OperationMetadata`). But I trust our customers to find the `use` declarations.